doc/api/fs.md

```diff
@@ -797,8 +797,8 @@ the end of the file.
 
 _Note: The behavior of `fs.open()` is platform specific for some flags. As such,
 opening a directory on OS X and Linux with the `'a+'` flag - see example below -
-will return an error. Whereas on Windows and FreeBSD a file descriptor will be
```

Comment:
I considered that, but then you have a fairly long sentence that is probably harder to understand than the two separate sentences:

> As such, opening a directory on OS X and Linux with the `'a+'` flag - see example below - will return an error, whereas on Windows and FreeBSD a file descriptor will be returned.

I could be persuaded, but I do prefer splitting that into two sentences. The `see example below` in the middle of the long sentence is especially kind of begging for a shorter sentence.